sql/driver/wire.go

```diff
@@ -112,11 +114,9 @@ func (d Datum) Value() (driver.Value, error) {
 	return val, nil
 }
 
-// GoTime returns the receiver as a time.Time in UTC. It is critical
-// that the time.Time returned is in UTC, because that is the
-// storage/wire contract for dates and times.
+// GoTime returns the receiver as a time.Time.
 func (t Datum_Timestamp) GoTime() time.Time {
-	return time.Unix(t.Sec, int64(t.Nsec)).UTC()
+	return time.Unix(t.Sec, int64(t.Nsec))
```

Comment:
What's the value in moving the call to `UTC()` to the caller? I don't see any harm, just wondering what the value is here.